src/chat.rs

```diff
@@ -2796,7 +2796,8 @@ pub(crate) async fn add_info_msg(context: &Context, chat_id: ChatId, text: impl
             text.as_ref().to_string(),
             rfc724_mid,
         ]
-    ).await.is_err() {
+    ).await {
+        error!(context, "Could not add info msg: {}", e);
```

Comment:
are you aware that `error!(...)` shows up at the UI as a toast?

Reply:
I was once, but I forgot

Reply:
but is this right here? can the user react in any way to this error?  If not, it should be a warning so that developers see that if a user manages to submit a log.

Reply:
"I was once, but I forgot" was supposed to mean "Oops, fixed this"